tests/python/rest_api/_test_base.py

```diff
@@ -21,6 +22,14 @@
 from shared.utils.config import make_api_client
 from shared.utils.helpers import generate_image_files, generate_video_file
 
+SHARE_DIR = Path(os.path.dirname(__file__), "../../mounted_file_share").resolve()
```

Comment:
Seems like it would be useful to have this available globally, in `tests/python/shared/utils/config.py`.